tests/Goteo/Model/FaqTest.php

```diff
@@ -119,15 +149,32 @@ public function testDelete(Faq $ob): Faq
      * @depends testDelete
      */
     public function testNonExisting(Faq $ob) {
+        $this->expectException(ModelNotFoundException::class);
         $ob = Faq::get($ob->id);
-        $this->assertFalse($ob);
-        $this->assertFalse(Faq::remove($ob->id));
     }
 
     /**
      * Some cleanup
      */
     static function tearDownAfterClass(): void {
         delete_test_node();
+        self::delete_faq_section();
     }
+
+    private static  function delete_faq_section(): bool
+    {
+        try {
+            $faqSection= FaqSection::getBySlug(self::$sectionData['slug']);
+            $faqSection->dbDelete();
+        }
+        catch(ModelNotFoundException $e) {
+        }
+        catch(\PDOException $e) {
+            error_log('PDOException on deleting test faq section! ' . $e->getMessage());
```

Comment:
If there's an error while removing the FaqSection, the test should simply fail.

There's no need to pollute the PHPUnit logs. Right now the shows the message between the test execution and is not clear what's going on.

Tests shouldn't print anything.